lib/spack/spack/cmd/view.py

```diff
@@ -157,11 +164,23 @@ def view(parser, args):
     specs = spack.cmd.parse_specs(args.specs)
     path = args.path[0]
 
+    if args.projection_file:
+        if os.path.exists(args.projection_file):
+            with open(args.projection_file, 'r') as f:
+                projections_data = s_yaml.load(f)
+                validate(projections_data, spack.schema.projections.schema)
+                ordered_projections = projections_data['projections']
+        else:
+            tty.error('Specified projection file does not exist.')
```

Comment:
I think it's more appropriate to die in this context, since the command explicitly specified a file that doesn't exist. I think anything else will be a silent change in what the user expects. I've updated it to die instead of using `tty.error`